libs/designsystem/src/lib/components/accordion/accordion-item.component.scss

```diff
@@ -27,7 +27,7 @@ $padding: utils.size('s');
 }
 
 .kirby-icon {
-  transition: transform utils.get-transition-duration('quick');
+  transition: transform interaction-state.$default-transition-duration;
```

Comment:
Not sure we should use it here. It's not related to interaction states is it?

Reply:
Good point, I was a bit quick here. Reverting!